### PR TITLE
Added week goals

### PR DESCRIPTION
**Pull request by Rogggger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,28 @@
 # mini-C-Interpreter
 Developed using C++ by team 端茶倒水 in BIT
+
+## 本周任务 *Edit September 1st*
```
Comment (chengdazhi) [inline README.md]: 可以把本周任务改成8.28-9.3，方便以后继续添加每周任务

**Comment by Rogggger** [comment]
添加完了，再看下

**Comment by chengdazhi** [comment]
Looks Good!

**Comment by chengdazhi** [comment]
You can choose to delete the deprecated branch.

**Pull request merged**